fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -646,58 +640,10 @@ public LiveData<SingleEvent<String>> getScanWarningEvent() {
     return mScanWarningEvent;
   }
 
-  public Election getCurrentElection() {
-    try {
-      return electionRepo.getElection(laoId, currentElection);
-    } catch (UnknownElectionException e) {
-      return null;
-    }
-  }
-
-  public void setCurrentElection(String electionId) {
-    currentElection = electionId;
-  }
-
-  public MutableLiveData<List<Integer>> getCurrentElectionVotes() {
-    return mCurrentElectionVotes;
-  }
-
-  public RollCall getCurrentRollCall() {
-    return currentRollCall;
-  }
-
-  public void setCurrentRollCall(RollCall rc) {
-    currentRollCall = rc;
-  }
-
   public void setCurrentRollCallId(String rollCallId) {
     currentRollCallId = rollCallId;
   }
```

Comment:
Is this still necessary ?

Reply:
Actually, it's still being used.

I'll create a commit to apply the same idea of "the fragment knows the rollcall id, not the viewmodel"

Reply:
Actually, it is used by the QRScanningFragment.

And as this is a big chunk of badly modularized code, I need to keep this for now.